Doxyfile

```diff
@@ -0,0 +1,23 @@
+PROJECT_NAME           = "OpenROAD's Doxygen Pages"	
+PROJECT_BRIEF          = "Compilation of detailed documentation for OpenROAD modules"
+PROJECT_LOGO           = ""
+OUTPUT_DIRECTORY       = "./_readthedocs/html/doxygen_output"
+DISTRIBUTE_GROUP_DOC   = YES
+EXTRACT_ALL            = YES	
+EXTRACT_PRIVATE        = YES
+EXTRACT_PACKAGE        = YES
+EXTRACT_STATIC         = YES
+EXTRACT_ANON_NSPACES   = YES
+INPUT                  = ./src/odb/include/odb/db.h ./docs/doxymain.md 
```

Comment:
Do we have to list all files here? Is there an option for recursively searching `src/**`?

Reply:
Recursive takes too long as I stated above. (More than 30 mins)